yt/yql/agent/yql_agent.cpp

```diff
@@ -378,8 +378,9 @@ class TYqlAgent
             auto refreshTokenExecutor = New<TPeriodicExecutor>(ControlInvoker_, BIND(&RefreshToken, user, token, queryClients), Config_->RefreshTokenPeriod);
             refreshTokenExecutor->Start();
 
+            const THashMap<TString, THashMap<TString, TString>> credentials = {{"yt_default", {{"category", "yt"}, {"content", token}}}};
```

Comment:
We are not adding any new credentials here, we are simply changing the way existing ones are provided. Will this be implemented in the next PR?